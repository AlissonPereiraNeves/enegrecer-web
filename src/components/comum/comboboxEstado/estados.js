import estadosJSON from './estados.json';

export default estadosJSON.map(estado => estado.sigla).sort((a, b) => {
  if (a < b) {
    return -1;
  }
  if (a > b) {
    return 1;
  }
  // names must be equal
  return 0;
});
