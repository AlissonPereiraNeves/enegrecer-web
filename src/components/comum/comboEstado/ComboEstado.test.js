import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Estado from '../comboEstado'
import estados from './estados'

describe('Combo Estado', () => {
    it('nÃ£o deve possuir estrutura definida ', () => {
      const onChange = jest.fn();
      const estado = mount(<Estado id={'13'} onChange={onChange} />);
      const estadoTextoJson = toJson(estado);
      expect(estadoTextoJson).toMatchSnapshot();
    });
});