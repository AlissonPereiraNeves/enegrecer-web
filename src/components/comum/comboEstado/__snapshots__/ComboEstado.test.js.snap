// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Combo Estado n√£o deve possuir estrutura definida  1`] = `
<comboEstado
  id="13"
  onChange={[Function]}
>
  <Combobox
    divClasse=""
    id="13"
    itens={Array []}
    label=""
    onChange={[Function]}
    valorPadrao=""
  >
    <div
      className=""
    >
      <select
        id="13"
        onChange={[Function]}
      >
        <option
          value=""
        />
      </select>
      <label
        htmlFor="13"
      />
    </div>
  </Combobox>
</comboEstado>
`;
